### ISSUE TITLE
EMA model for backpropagation gradient Descent ?

### ISSUE THREAD

**Issue by gndlwch2w** [VivienLu/UPCoL#3]
Dear author, 
    Thank you for your excellent work, but I am confused that the loss `loss_pc_unlab` calculated on the EMA model for backpropagation gradient descent ?
